.github/workflows/gh-pages.yml

```
@@ -0,0 +1,83 @@
+name: Build for Github Pages
+
+on:
+  push:
+    branches: dev
+
+jobs:
+  build-test:
+    runs-on: ubuntu-latest
+
+    environment: dev
+
+    strategy:
+      matrix:
+        node-version: [22.x]
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Cache node modules
+        id: cache-npm
+        uses: actions/cache@v4
+        env:
+          cache-name: cache-node-modules
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
+
+      - name: Install Bower and Grunt
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ matrix.node-version }}
```

This would replace the previous "Cache node modules" step.
```suggestion
          node-version: ${{ matrix.node-version }}
          cache: npm
```

---

@nschonni Yeah, I was trying some stuff as this solution was still taking a lot of time to install dependencies. After many trial and errors, I realized that the issue was with WET-BOEW and not the dependencies installation. I have reverted back to using `cache: npm`. I will be working on changing the "prepare" action from WET-BOEW as it runs many unnecessary tasks unrelated to building the dist files which take a lot of time. It should reduce the installation time by about 2-2.5min.